db/python/layers/web.py

```diff
@@ -15,15 +16,10 @@
 from db.python.tables.base import DbBase
 from db.python.tables.project import ProjectPermissionsTable
 from db.python.tables.sequencing_group import SequencingGroupTable
-from models.models import (
-    AssayInternal,
-    FamilySimpleInternal,
-    NestedParticipantInternal,
-    NestedSampleInternal,
-    NestedSequencingGroupInternal,
-    SearchItem,
-    parse_sql_bool,
-)
+from models.models import (AssayInternal, FamilySimpleInternal,
```

Comment:
This is a fun linting one, soon to be resolved in metamist, but your default isort profile in VSCode is incorrect, in your VSCode settings.json you can add:

```json
    "isort.args": [
        "--profile=black"
    ],
```

We intend to add this to metamist's pyproject.toml soon (to replicate https://github.com/populationgenomics/cpg-python-template-repo/blob/6e2da21e1012e6d2b15ba0c385da579a9b80b550/pyproject.toml#L7), but we were waiting for other big PRs to be finalised before introducing lots of merge conflicts.

Reply:
Reverted and fixed. I assume that replication to metamist will happen along with the new `pre-commit` hooks?